.github/workflows/ort.yml

```diff
@@ -0,0 +1,189 @@
+
+name: The OSS Review Toolkit (ORT)
+
+on:
+    schedule:
+      - cron: "0 0 * * *"
+    push:
+        tags:
+            - "ort-for-v*.*"
```

Comment:
do you want us to create tags specifically for this? is there no other mechanism for manual start?

Reply:
changed